app/controllers/projects_controller.rb

```diff
@@ -48,6 +48,6 @@ def update
   private
 
   def project_params
-    params.require(:project).permit(:group_members, :name, :project_type, :final_confirmation, :demo_night_id, :note).merge(user_id: current_user.id)
+    params.require(:project).permit(:group_members, :name, :project_type, :final_confirmation, :demo_night_id, :note, :description).merge(user_id: current_user.id)
```

Comment:
unrelated to your change, but why the fuck are we merging the user_id in!?

Reply:
Interesting. I think this is safer than adding a hidden field to the form. Would you feel better about it if it were in the `create` explicitly?